### ISSUE TITLE
flowcrypt.com/feed is redirects to http://127.0.0.1/blog/page/feed/

### ISSUE THREAD

**Issue by seisvelas** [FlowCrypt/flowcrypt-docs#66]
While messing around with some stuff in the Firefox extension, I noticed that some requests were being made to /feed, so I browsed there and was redirected to http://127.0.0.1/blog/page/feed/.

Perhaps this works locally, or used to? I haven't debugged this at all yet, since I noticed while I'm still trying to do something else. But I made this issue so I don't forget to look at why this is happening / fix it.

**Comment by tomholub** [comment]
We had a few issues like that, with a few pages, and the fix belongs somewhere in nginx config. Likely nginx-leader, but possibly nginx-web.

**Comment by seisvelas** [comment]
What should it be doing? I notice that https://flowcrypt.com/blog/page/feed/ is a blank page anyway. We have an RSS feed on https://flowcrypt.com/blog/feed.xml, should this be loading that?

**Comment by tomholub** [comment]
Then it should be probably showing a 404

**Comment by tomholub** [comment]
It would be interesting though to find out exactly why are these redirects to localhost happening, because I feel like I had to fix them one by one, instead of fixing it in a more global manner.

**Comment by seisvelas** [comment]
>Then it should be probably showing a 404

True, but the browser extension must *think* something is there, or it wouldn't make the request. If we find out what the extension thinks is supposed to be there, we can either serve it on this broken page, or tell the extension to look somewhere else.

**Comment by tomholub** [comment]
Sorry, I missed this. Yes! `flowcrypt.com/feed` does not sound correct because it's a blog feed after all, so `https://flowcrypt.com/blog/feed.xml` sounds more appropriate. There are therefore two fixes: one is that the browser extension should be calling`https://flowcrypt.com/blog/feed.xml`. 

The second one is that we should not be redirecting to 127.0.0.1. Not just this particular address, but in general. If you could figure this one out, that would be very helpful - I wasn't able to look into this successfully in the past.
